webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -1161,9 +1160,31 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
 
     Future
       .sequence(
-        listNodesThatShouldExist
-          .map(listNodeIri => ResourceUtilV2.checkListNodeExists(listNodeIri, appActor))
-          .toSeq
+        listNodesThatShouldExist.map { listNodeIri =>
+          for {
+            checkNode <- ResourceUtilV2.checkListNodeExistsAndIsRootNode(listNodeIri, appActor)
+
+            _ = checkNode match {
+                  // it doesn't have isRootNode property - it's a child node
+                  case Right(false) =>
+                    FastFuture.successful(())
```

Comment:
you are not interessted in the value, so simply return `()`. No need to warp it into a Future only to discard it later.

Reply:
Same as above - I went for the same solution as I implemented in `ValueResponderV2`. but w/o`FastFuture.successful(())` it didn't compile. Strangely now it seems working. Suspect metals was broken at that time...